pkg/controller/utils/clusterinfo.go

```diff
@@ -101,3 +157,15 @@ func getInfrastructureObject(kclient client.Client) (*configv1.Infrastructure, e
 	}
 	return infra, nil
 }
+
+// AWSOwnerTag returns owner taglist for the cluster
+func AWSOwnerTag(kclient client.Client) (map[string]string, error) {
+	m := make(map[string]string)
+	name, err := GetClusterName(kclient)
+	if err != nil {
+		return m, err
+	}
+
+	m[fmt.Sprintf("kubernetes.io/cluster/%s", name)] = "owned"
+	return m, nil
+}
```

Comment:
this is the cluster default tag.  Looks good

Reply:
Yup, it surely is! :)